vendors/swedbank-pay-core-library/tests/bootstrap.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+if (file_exists(__DIR__ . '/../vendor/autoload.php')) {
+    require_once __DIR__ . '/../vendor/autoload.php';
+} else {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ["require_once" statement detected. File manipulations are discouraged. Concatenating is forbidden.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)